PowerPointLabs/PowerPointLabs/PositionsLab/PositionsPaneWPF.xaml.cs

```diff
@@ -370,12 +370,11 @@ void _leftMouseUpListener_Rotation(object sender, SysMouseEventInfo e)
 
         void _leftMouseDownListener_Rotation(object sender, SysMouseEventInfo e)
         {
-            try
-            {
-                ToggleButton button = ((bool)rotationButton.IsChecked) ? rotationButton :
+            ToggleButton button = ((bool)rotationButton.IsChecked) ? rotationButton :
                              ((bool)duplicateRotationButton.IsChecked) ? duplicateRotationButton
```

Comment:
This code might not be modified in your PR, but I feel that instead of assigning null to button, it may be better to return straight? Because calling button.IsMouseOver on null knowing that it will be null feels weird. (But it seems the previous author intended for this because an error will be logged when button gets set to null, seems to promote code reuse in a rather unusual way).